dtfs-central-api/src/drivers/db-client.js

```diff
@@ -37,9 +39,9 @@ const getClient = async () => {
 module.exports.getClient = getClient;
 
 /**
- * @template TDoc
- * @param {DbCollectionName} collectionName
- * @return {Promise<import('mongodb').Collection<TDoc>>}
+ * @template {DbCollectionName} TCollectionName
+ * @param {TCollectionName} collectionName
+ * @return {Promise<import('mongodb').Collection<DbModel<TCollectionName>>>}
```

Comment:
This change, using a new `DbModel` I've created, means that when we fetch a document (or documents) from the data base it will use the `collectionName` you specify to work out what the shape of that object will be. e.g. if we do
```
db.getCollection('banks');
```
then any documents we find off the back of any queries on that collection will have the `Bank` type from `types/db-models/banks.d.ts`.